src/sentry/migrations/0529_new_notification_tables.py

```diff
@@ -0,0 +1,128 @@
+# Generated by Django 3.2.20 on 2023-08-14 23:19
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.db.models.fields.hybrid_cloud_foreign_key
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0528_truncate_flat_index"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="NotificationSettingProvider",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("scope_type", sentry.db.models.fields.bounded.BoundedPositiveIntegerField()),
+                ("scope_identifier", sentry.db.models.fields.bounded.BoundedBigIntegerField()),
```

Comment:
If you can, using string enums will make adhoc querying and debugging easier as you don't have to keep the context of what enum int value maps to which type in your head.

Reply:
@markstory fixed to use strings